tests/data/__init__.py

```diff
@@ -22,6 +22,9 @@
 backup_version_bcfdad6b7355_1 = CWD / "backups/backup_version_bcfdad6b7355_1.zip"
 """bcfdad6b7355: remove tool name and slug unique contraints"""
 
+backup_version_09aba125b57a_1 = CWD / "backups/backup_version_09aba125b57a_1.zip"
+"""09aba125b57a: add OIDC auth method (Safari-mangled ZIP structure)"""
```

Comment:
This might be a silly question, but what's the relevance of `add OIDC auth method`?? 😕
Appears in a couple files.

Reply:
The original purpose of having a bunch of example backup files in the tests data was to test for migration issues, so for all of the migration test data I include the alembic version (`09aba125b57a` in this case) as well as the migration message. This is basically saying "this backup is a snapshot of a database that is at the migration where the OIDC auth method was added".

For this specific test it's kind of useless, though in the future if this migration fails it's easier to debug

Reply:
Ah, makes sense. I tried to check if it was a commit hash and obviously it wasn't! Didn't think of alembic